docs/framework/install/net-framework-3-5-on-windows-8-plus.md

```diff
@@ -19,52 +19,56 @@ ms.author: "mairaw"
 manager: "wpickett"
 ---
 
-# Installing the .NET Framework 3.5 on Windows 8, Windows 8.1 and Windows 10
-The .NET Framework is an integral part of many applications running on Windows and provides common functionality for those applications to run. For developers, the .NET Framework provides a consistent programming model for building applications. if you are using the Windows operating system, the .NET Framework may already be installed on your computer. Specifically, the [!INCLUDE[net_v45](../../../includes/net-v45-md.md)] is included with [!INCLUDE[win8](../../../includes/win8-md.md)], the [!INCLUDE[net_v451](../../../includes/net-v451-md.md)] is included with [!INCLUDE[win81](../../../includes/win81-md.md)] and the [!INCLUDE[net_v46](../../../includes/net-v46-md.md)] is included with Windows 10.  
+# Installing the .NET Framework 3.5 on Windows 8, Windows 8.1, and Windows 10
```

Comment:
we should avoid ing in titles. also the title in the metadata for this one looks weird. seems to be in the wrong topic.